packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -1202,16 +1202,16 @@ export class PaginatedMessage {
 	public addPageAction(action: PaginatedMessageAction, index: number): this {
 		if (index < 0 || index > this.pages.length - 1) throw new Error('Provided index is out of bounds');
 
-		if (!this.pageActions[index]) {
-			this.pageActions[index] = new Map<string, PaginatedMessageAction>();
-		}
+		const pageActionAtIndex = this.pageActions[index] ?? new Map<string, PaginatedMessageAction>();
```

Comment:
Optionally this, but lgtm

```suggestion
		const pageActionAtIndex = (this.pageActions[index] ??= new Map<string, PaginatedMessageAction>());
```